test-framework/src/test/java/org/apache/brooklyn/test/framework/TestSshCommandTest.java

```diff
@@ -30,6 +30,8 @@
 import static org.apache.brooklyn.test.framework.TestSshCommand.DOWNLOAD_URL;
 import static org.apache.brooklyn.test.framework.TestSshCommand.SHELL_ENVIRONMENT;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.apache.brooklyn.core.entity.EntityAsserts.assertServiceFailed;
```

Comment:
This name doesn't match the new method names you've used - `assertEntityHealthy` etc.

Presumably you'd also need to update the calling code.